src/main/groovy/com/vanniktech/android/junit/jacoco/GenerationPlugin.groovy

```diff
@@ -28,6 +28,13 @@ class GenerationPlugin implements Plugin<Project> {
                     addJacoco(subProject, extension, mergeTask, mergedReportTask)
                 }
             }
+
+            rootProject.afterEvaluate {
```

Comment:
The problem is that the Jacoco version is read from extension above. But this happens when the plugin is applied what is too early. The user didn't have a chance to modify the Jacoco version, yet. The fix applies the Jacoco version again after the extension has been initialized in the build scripts.

I didn't find a way to test fix. No test is using this method.